README.md

```diff
@@ -73,6 +73,16 @@ Follow the steps below to update your core files.
 
 ## FAQ
 
+### What’s the difference between this project and drupal/recommended-project?
+
+The two most commonly recommended Drupal project templates are this project ([drupal-composer/drupal-project](https://github.com/drupal-composer/drupal-project)) and [drupal/recommended-project](https://github.com/acquia/drupal-recommended-project).
```

Comment:
The link for drupal-recommended-project is wrong. acquia => drupal

Reply:
D'oh, yeah, I was working on acquia/drupal-recommended-project at the same time, we'd like to base it on one of these two upstream templates. I'll fix the link once we come to consensus on the wording.